evalml/pipelines/multiseries_regression_pipeline.py

```diff
@@ -133,7 +161,14 @@ def predict_in_sample(
             objective,
             calculating_residuals,
         )
-        stacked_predictions = stack_data(unstacked_predictions)
+        if include_series_id:
+            stacked_predictions = stack_data(
+                unstacked_predictions,
+                include_series_id=True,
+                series_id_name=self.series_id,
+            )
+        else:
+            stacked_predictions = stack_data(unstacked_predictions)
```

Comment:
We shouldn't need to split these into separate logical calls - if we pass in the `series_id_name` when `include_series_id=False`, nothing will happen. This can be condensed to
```
stacked_predictions = stack_data(
    unstacked_predictions,
    include_series_id=include_series_id,
    series_id_name=self.series_id
)
```